apps/docs/src/content/docs/libraries/db.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: db
+description: Database interactor with DrizzleORM.
+---
+
+## 📦 Overview
+
+### What it does
+
+This library holds the schemas and commands for making and using the database using DrizzleORM and Drizzle-Kit.
```

Comment:
You can hyperlink 'DrizzleORM' and 'Drizzle-Kit', saves people a bit of time if they're interested in learning more.